### DISCUSSION TITLE
WIP: Unmerged bars?

### DISCUSSION THREAD

**Discussion by garrett** [cockpit-project/cockpit-files#474] [Show and tell]
@allisonkarlitskaya was asking about what it would be like to unmerge the action bars.

Instead of:

![Cockpit FIles, with inline breadcrumbs](https://github.com/cockpit-project/cockpit-files/assets/10246/c36eba04-fe02-4830-a284-5eedc93a1e42)

It would look like this:

![Cockpit FIles, with inline breadcrumbs(1)](https://github.com/cockpit-project/cockpit-files/assets/10246/f2f0330a-87cf-43e8-b3dd-deebab6cfd72)

---

Note, this has nothing to do with mobile, as mobile with either would look something like this:

![Navigator, with inline breadcrumbs](https://github.com/cockpit-project/cockpit-files/assets/10246/2de2591d-329d-47f1-8e3b-80ecf53d2778)

**Comment by allisonkarlitskaya** [comment]
If the kebab menu contains global settings type stuff then maybe it would make sense "upstairs" still?

**Reply by garrett** [reply]
Right, but we won't have anything for the global menu at the moment, so I removed it from the mockup for now.

**Reply by garrett** [reply]
However, I do have to say it looks a little silly with both ︙ menus.

![Navigator, with inline breadcrumbs(1)](https://github.com/cockpit-project/cockpit-files/assets/10246/b7ca5b35-56df-4553-b531-6c02e4d447da)

**Reply by garrett** [reply]
Quick mod with the toolbar more obviously in the content area, and it's better in mobile:

![Navigator, with inline breadcrumbs(4)](https://github.com/cockpit-project/cockpit-files/assets/10246/43854f86-5e70-4d9f-94a6-05db8e4cd336)

I dropped the margins on the sides too, which is what we usually do in mobile mode.
